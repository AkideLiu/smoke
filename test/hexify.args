fixtures/hexify
fixtures/hexify.rb
