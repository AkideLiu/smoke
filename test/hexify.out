sentence
[32m  succeeded[0m
single-character
[32m  succeeded[0m
word
[32m  succeeded[0m

[32m3 tests, 0 failures[0m
